meson.build

```diff
@@ -139,6 +139,10 @@ endif
 if get_option('enable-blas')
   extra_defines += '-DUSE_BLAS=1'
 
+  if get_option('enable-neon')
```

Comment:
If we are going to use neon in arm architecture only then we need to check that platform is android

Reply:
neon is only available in arm architecture. it also should work in tizen platform in raspberry pi.